.github/workflows/deploy.yaml

```diff
@@ -33,11 +33,24 @@ jobs:
           S3BucketName=vapor-blog-site,
           AcmCertificateArn=${{ secrets.CERTIFICATE_ARN }}
       if: steps.cred.outcome == 'success'
-    - name: Deploy to S3
+    - name: Deploy Main Site to S3
       id: s3deploy
       uses: jakejarvis/s3-sync-action@master
       with:
-        args: --acl public-read --follow-symlinks --delete
+        args: --acl public-read --follow-symlinks --delete --exclude 'feed.rss'
+      env:
+        AWS_S3_BUCKET: 'vapor-blog-site'
+        AWS_ACCESS_KEY_ID: ${{ secrets.BLOG_DEPLOYER_AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.BLOG_DEPLOYER_AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: 'eu-west-2'
+        SOURCE_DIR: 'Output'
```

Comment:
Looks good, but FYI this could be rather trivially converted to direct aws cli calls, and it would look even simpler because you don't need to pass envs like that to another action.
It looks like [this](https://github.com/vapor/penny-bot/blob/1401f6c0ba504219886d0952540110d2db033f66/.github/workflows/deploy-all-lambdas.yml#L43) in Penny.